data/duke.txt

```diff
@@ -0,0 +1,16 @@
+E|true|project meeting|2019-10-15
```

Comment:
Perhaps, you could add this file into git ignore as we do not need to push this data.